shared/src/commonTest/kotlin/com/example/tasky/auth/domain/util/StringExtensionTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.tasky.auth.domain.util
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import kotlin.test.Test
+
+class StringExtensionTest {
+    @Test
+    fun `test getAvatarDisplayName 2 word`() {
+        val name = "abc def"
+        val actual = name.getAvatarDisplayName()
+
+        assertThat(actual).isEqualTo("ad")
+    }
```

Comment:
These are the type of tests I was talking about in [this comment](https://github.com/2013612/Tasky/pull/175#discussion_r1859473329) 😄